src/test/librbd/test_snapshot_flatten_rollback_cloned.py

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env python
```

Comment:
Can you move this test to `test_rbd.py`?  Probably enough to just verify parent spec after rollback.  Would also be good to add tests for various combinations of updates (no parent->parent, parent->no parent, overlap size change, etc).